assets/app/styles/_list.less

```diff
@@ -0,0 +1,22 @@
+.confirm-delete {
+  background: @state-warning-bg;
+  color: @state-warning-text;
+  margin-top: 0px;
+  padding: 10px 25px;
+  position: relative;
+}
+
+.delete-button {
```

Comment:
seems like this is a pretty generic style that we will reuse, rename to .action-icon and move to _core.less